vault/core.go

```diff
@@ -1955,6 +1967,16 @@ func (c *Core) SetLogLevel(level log.Level) {
 	}
 }
 
+func (c *Core) SetConfig(conf *server.Config) {
```

Comment:
It doesn't look like this gets called for test clusters.  Should it be?

Reply:
This is only called when the config is reloaded on SIGHUP, similar to `core.SetLogLevel`. We set the config for test cluster cores in here: https://github.com/hashicorp/vault/pull/7424/files#diff-359912a7f1b2eaf50ca9cfd666c79dabR1350

Reply:
Ah, I misunderstood SetConfig's purpose I see.  I don't think the code you reference is sufficient to populate Core.rawConfig though.  For one, base may be nil, and even if it's non-nil base.RawConfig is probably going to be nil.

Reply:
Should be addressed in 9cfbdb3e20fc6275473fee66a784652c2fda01dd.